### DISCUSSION TITLE
Update config

### DISCUSSION THREAD

**Discussion by pott3r3r** [blakeblackshear/frigate#6280] [Ask A Question] [answered]
Hello community!
So I've found myself in a hole, thats been dug over many years. I have somewhat a working config, but saw a youtube video of all the updates, so decided to update. Main issue here is prolly that my config from start was not perfect. I run nvidia-docker.
that is the current stuff in docker-compose

```frigate:
    container_name: frigate
    privileged: true # this may not be necessary for all setups
    restart: unless-stopped
    image: ghcr.io/blakeblackshear/frigate:stable
    runtime: nvidia
    environment:
      - NVIDIA_VISIBLE_DEVICES=all
      - NVIDIA_DRIVER_CAPABILIIES=compute,utility,video
    devices:
      - /dev/dri/renderD128 # for intel hwaccel, needs to be updated for your hardware
      - /dev/dri/card0
    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/docker/frigate/config:/config
      - /home/
docker/frigate/media:/media
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 4000000000
    ports:
      - "8554:8554"
      - "5000:5000"
      - "1935:1935" # RTMP feeds
```

[frigatelog.txt](https://github.com/blakeblackshear/frigate/files/11340539/frigatelog.txt)
[newconfig.txt](https://github.com/blakeblackshear/frigate/files/11340540/newconfig.txt)
[oldworkingconfig.txt](https://github.com/blakeblackshear/frigate/files/11340541/oldworkingconfig.txt)

**Comment by NickM-27** [comment]
Please make a support issue

**Reply by pott3r3r** [reply]
No idea where that is found, I went by the "ask community for help"-option?

**Reply by NickM-27** [reply]
https://github.com/blakeblackshear/frigate/issues

**Reply by pott3r3r** [reply]
![image](https://user-images.githubusercontent.com/43346681/234870839-cfa3ac60-e1ae-454e-8cbf-b1eef365b6ad.png)
just love computers atm

**Reply by NickM-27** [reply, marked as answer]
your frigate error logs are repeated many times, only one copy is needed

**Reply by pott3r3r** [reply]
Sorry, I'm a horrible person :'(
